docs/howto/operate/move-hub.md

```diff
@@ -106,6 +110,53 @@ do your modifications. This prevents needing to create another EC2
 instance just for this.
 ```
 
+### Azure Files
+
+```{note}
+The exception: even if the source and target cluster are in the same Azure subscription, re-using the same home directory storage might prove more difficult than copying files from one storage to the other.
```

Comment:
This note just at the beginning of the Azure Files section feels a little bit weird... I mean, it seems the result of some previous paragraph discussing something and that paragraph does not exist above, AFAIK.
I am missing something, maybe?

Reply:
Yeah, I'd maybe rephrase it as "You may be tempted to attach an existing NFS server across two clusters. However, this is not possible since Azure VMs cannot simultaneously exist in two virtual networks and each cluster will have it's own network."

Reply:
Great rephrasing @sgibson91, thanks! I will use it, if that's ok with you :D 

> This note just at the beginning of the Azure Files section feels a little bit weird... I mean, it seems the result of some previous paragraph discussing something and that paragraph does not exist above, AFAIK.
I am missing something, maybe?

@damianavila, I was refering to the paragraph [here](https://github.com/2i2c-org/infrastructure/blob/master/docs/howto/operate/move-hub.md#copy-home-directories):
*"This might not entirely be necessary - if the source and target cluster
are in the same GCP Project / AWS Account, we can just re-use the same
home directory storage!"*

But now I see I didn't do a good job capturing that idea :sweat_smile: I will go with Sarah's rephrasing, since that's much better.

Thank you! :sparkles: